repository/src/main/java/org/apache/atlas/repository/store/graph/v1/SoftDeleteHandlerV1.java

```diff
@@ -70,6 +70,7 @@ protected void deleteEdge(AtlasEdge edge, boolean force) throws AtlasBaseExcepti
         if (LOG.isDebugEnabled()) {
             LOG.debug("==> SoftDeleteHandlerV1.deleteEdge({}, {})",GraphHelper.string(edge), force);
         }
+        boolean isRelationshipEdge = isRelationshipEdge(edge);
```

Comment:
why assigning to a variable ? Plz rewrite as below , as I dont see any usage of the variable
```
if (isRelationshipEdge(edge)) {
}
```

Reply:
So checking `isRelationshipEdge(edge)` after deleting the edge causing IllegalStateException: edge not found. So we've extracted the value first.